components/automate-chef-io/content/docs/backup.md

```diff
@@ -188,69 +188,81 @@ where `bucket_name` is the name of the S3 bucket `base_path` an optional path wi
 
 Restore Chef Automate to an instance on which Automate is not already installed.
 
-### Restore From a Filesystem Backup
-
-#### Prerequisites
+### Prerequisites
 
 1. On the restore host, download and unzip the Chef Automate command-line tool:
-    ```shell
+
+   ```shell
         curl https://packages.chef.io/files/current/latest/chef-automate-cli/chef-automate_linux_amd64.zip | gunzip - > chef-automate && chmod +x chef-automate
     ```
 
-1. Ensure access for Chef Automate to a backup directory in the [configured location]({{< ref
-"backup.md#backup-to-filesystem" >}}):
+1. *(filesystem backups only)* Ensure access for Chef Automate to a backup directory in the [configured location]({{< ref
```

Comment:
I don't love the italics here; happy to go with whatever convention we use to emphasize text.

Reply:
Slight change to wording & formatted with bold. Italics are the usual convention for emphasis, but they're not as visible on the screen as they are in print.